src/index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>2048</title>
+    <title>2048 - Now as TypeScript</title>
```

Comment:
Are you sure this is what we should be using for the title?  I'm not sure we should be advertising which language we used to make the game.

Reply:
Why not? TS nerds love it

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;This comment was created from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=d0c650b8-284f-403c-b4fd-e2f1ae69cfe0).

Reply:
I'm just not sure anyone playing the game will care about it!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;This comment was created from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=93a56271-3770-4ef9-b400-7f4940419562).

Reply:
I agree, but disagree!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=c98d3e5e-c150-4afe-9171-60c692ebaa12).

Reply:
Are you sure?

Reply:
I feel pretty strongly about this!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=9baa3b84-484c-43c7-acc1-98cf2551b3a1).

Reply:
Aww 😭

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=0f387fca-a512-4411-a4fd-a74300461bcc).

Reply:
Hello!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=dbee993e-5fef-433b-a40b-173f7cad8702).

Reply:
Too bad! I don't care!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=5a7c9b91-c794-45e7-9a0a-2d968d9e1d1b).

Reply:
Messaging!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=8e48e344-a3e4-4414-b5d7-a8c983e55f67).

Reply:
Ping ping ping!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=0b3ccf31-c100-4232-afe4-0c3695683ebc).

Reply:
Pong! 

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=eb9aec3d-2438-4f61-8a86-5b5b294ba8ec).

Reply:
Blambo!

Reply:
Yes, I'm sure!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=4c0faa95-8f33-4e4d-a514-0f86d4120212).

Reply:
OK -- we'll see if we get any feedback on this

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=11cc0b6c-c1de-4dc1-8d30-551a4a9fa65b).

Reply:
Blamo for John!

Reply:
Blammo!! 💥

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/2f6da8dc-bad3-495d-b269-4720f83f1002/thread/059d4475-42aa-4d35-a95d-8043ac02c39d?message=436ebac0-4df9-4775-9c0a-b293c2c4914c).

Reply:
This is a test for Nav!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/70121974-e0cd-45a5-900a-686a12c74905/thread/cf026bad-faf1-4234-a734-898a44ca91e3?message=879395a4-5a6d-4982-9337-ff83027fb9f2).

Reply:
Hello!

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/70121974-e0cd-45a5-900a-686a12c74905/thread/cf026bad-faf1-4234-a734-898a44ca91e3?message=6b080d2d-c6d0-4f5d-a373-3d954d586d9a).